javascript-methods/main.js

```diff
@@ -0,0 +1,77 @@
+const x = 10;
+const y = 8;
+const z = 7;
+
+const maximumValue = Math.max(x, y, z);
+console.log(maximumValue);
+const heroes = [1, 2, 3, 4];
+
+let randomNumber = Math.random();
+
+randomNumber *= heroes.length;
+
+const randomIndex = Math.floor(randomNumber);
+
+console.log(randomIndex);
+
+const randomHero = randomIndex[heroes];
+
+console.log(randomHero);
+
+// Array Methods
+
+// book objects
+
+const js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer',
+};
+const css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou',
+};
+
+const library = [
+  {
+    title: 'JS for intermidiate jobs',
+    author: 'Jack Almadjian',
+  },
+  {
+    title: 'CSS animations',
+    author: 'Alex Montoya',
+  },
+  {
+    title: 'Keyframes and 3d css styling',
+    author: 'shawn idk',
+  },
+];
+
+const lastBook = library.pop();
+
+console.log(lastBook);
+
+console.log(library[0]);
+
+// push method
+
+console.log(library.push(js));
+
+console.log(library.unshift(css));
+
+console.log(library.splice(1, 2));
```

Comment:
This should be `library.splice(1, 1)`. We only want to remove `1` element from the array, not `2`